overlay/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+
+include_directories(
+  "${BOOST_INCL}"
+  "${GLOG_INCL}"
+  "${PROTOBUF_INCL}"
+  "${PICOJSON_INCL}"
+  "${RAPIDJSON_INCL}"
+  "${PROCESS_INCL}"
+  "${STOUT_INCL}"
+  "${CURL_INCL}"
+  "${ZOOKEEPER_INCL}"
+  "${MESOS_ROOT}/build/include"
+  "${MESOS_ROOT}/include"
+  # FIXME: due to limitation of protobuf generation process
+  # and include in source
+  ${CMAKE_CURRENT_BINARY_DIR}/..
+  # FIXME: due to the following in agent.hpp
+  # #include <overlay/agent_metrics.hpp>
+  # #include <overlay/messages.hpp>
+  ${CMAKE_CURRENT_SOURCE_DIR}/..
+)
+
+add_library(overlay SHARED
+  agent_metrics.cpp
+  agent.cpp
+  # LNK2019: unresolved external symbol "public: __cdecl mesos:: log::Log::Log
+  # `log` is explicitly excluded for Windows in mesos
+  # master_metrics.cpp
+  # master.cpp
```

Comment:
Took me some time out that this comment refers to Mesos replicated log that is not supported on Windows; could you rephrase this comment to make this more clear?

Reply:
No. The above is clear to me. If you have a better way to make this more clear, give me some text and I will do replacement to avoid ping-pong.

Reply:
Found https://issues.apache.org/jira/browse/MESOS-5932 for porting replicated log to Windows (need to update leveldb first).

I would suggest, instead of putting error message into the comment, to point to the real underlying issue, like
```
# TODO(akornatskyy): Windows build of overlay module master (which depends on Mesos replicated log)
# can be re-enabled after replicated log is available on Windows (see MESOS-5932).
```

Reply:
done